base/redis/redis-cache.Deployment.yaml

```diff
@@ -64,7 +61,3 @@ spec:
             memory: 100Mi
       securityContext:
         runAsUser: 0
-      volumes:
```

Comment:
deploy-sourcegraph-docker repo has a volume/mount for redis-cache.

Reply:
Fixed in https://github.com/sourcegraph/deploy-sourcegraph-docker/commit/02c695926587e1ce9c01481ba675abc45d309e8e